### PR TITLE
Added Checkout time prop in ParkingRecord Entity and Added Vehicles c…

### PR DESCRIPTION
**Pull request by eswetharajesh**
Added checkOut Time property in the ParkingRecord entity and added VehiclesController code with Add, Delete, Overview(sort, search), Edit, Details, Park, UnPark actions.

### CODE REVIEW

File: Controllers/VehiclesController.cs [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+            catch (DbUpdateException ex)
+            {
+                // Log the error
+                ModelState.AddModelError("", "Unable to save changes. Try again, and if the problem persists, see your system administrator.");
+            }
+
+            PopulateDropDownLists(vehicle.VehicleTypeID);
+            return View(vehicle);
+        }
+
```
Comment (Kasleets) [inline Controllers/VehiclesController.cs:155]: Should we include more details of the vehicle in the 'Details' function?

**Review by jmc317 — APPROVED** [review]
Don't see any problems

**Review by Kasleets — APPROVED** [review]
It looks alright.

**Review by Xia-Hui-Liu — CHANGES_REQUESTED** [review]
can you remove all TempData["Message"], the message doesn't popup like last time. Will add alert from client-side

**Comment by Kasleets** [comment]
> can you remove all TempData["Message"], the message doesn't popup like last time. Will add alert from client-side

I believe we should keep TempData and remove it only one-by-one when we're ready with the implementation, not before

**Pull request merged**